src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 720px;
```

Comment:
maybe it can calculating with function?

Reply:
agree, you have all the variables for it